acid/acid3/numbered-tests.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<title>Acid3 numbered tests</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var tests = undefined;
+
+function gotMessage(e) {
+  var m = e.data;
+  if (tests === undefined && "tests" in m) {
```

Comment:
Any reason for the check on `tests`?

Reply:
Not really. Just me being paranoid not really understanding how all of the Acid 3 JS works. :)

Reply:
Check or not check seems fine with me. To really be paranoid, making something blow up if assumptions aren't true would be better.